crates/starknet-devnet-core/src/starknet/dump.rs

```diff
@@ -24,6 +25,20 @@ pub enum DumpEvent {
 }
 
 impl Starknet {
+    /// Create an instance of Starknet with the state generated from the events loaded from `path`.
+    pub fn load(config: &StarknetConfig, path: &str) -> DevnetResult<Self> {
+        let mut this = Self::new(config)?;
+
+        // Try to load transactions from dump_path, if there is no file skip this step
+        match this.load_events(path) {
+            Ok(events) => this.re_execute(events)?,
+            Err(Error::FileNotFound) => {}
```

Comment:
cant figure out why this error is not propagated. If a user makes a mistake with the path CLI arg he wont know about it. I know that this is old functionality. What do you think?

Reply:
Explanation added in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/571/commits/cb4caf5145f51f45b17be06bef41f5d2752c6da6